nix/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, python311Packages
+, pkgs
+, fetchFromGitHub
```

Comment:
@scientiac Do I understand well `pkgs.fetchFromGitHub` was tested but not used and the method for installing on NixOs is `$ git clone` and then install? Could you please add install instructions below the Install block in the file `Readme.MD` -  how to install this driver for NixOs? (Or the way is generally known as the usage of package managers for example? So do not have to be explicitly mentioned.. )

Reply:
Yes, I forgot to remove that one, I have removed it now.